azurecompute-arm/src/test/java/org/jclouds/azurecomputearm/features/StorageAccountApiLiveTest.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jclouds.azurecomputearm.features;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertFalse;
+import static org.testng.Assert.assertTrue;
+import static org.testng.Assert.assertNotNull;
+import com.google.common.collect.ImmutableMap;
+import org.jclouds.azurecomputearm.domain.CreateStorageServiceParams;
+import org.jclouds.azurecomputearm.domain.StorageService;
+import org.jclouds.azurecomputearm.domain.StorageServiceKeys;
+import org.jclouds.azurecomputearm.domain.StorageServiceUpdateParams;
+import org.jclouds.azurecomputearm.internal.BaseAzureComputeApiLiveTest;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+@Test(groups = "live", testName = "StorageAccountApiLiveTest")
+public class StorageAccountApiLiveTest extends BaseAzureComputeApiLiveTest {
+
+   private static final String NAME = String.format("%3.24s",
+           RAND + StorageAccountApiLiveTest.class.getSimpleName().toLowerCase());
+
+   private void check(final StorageService storage) {
+      assertNotNull(storage.id());
+      assertNotNull(storage.name());
+      assertNotNull(storage.storageServiceProperties());
+      assertNotNull(storage.storageServiceProperties().accountType());
+      assertFalse(storage.storageServiceProperties().primaryEndpoints().isEmpty());
+      assertNotNull(storage.storageServiceProperties().creationTime());
+   }
+
+   @Test()
```

Comment:
What I am wondering -
Without Rita's addition, the mvn test run lists (for me) 8 tests run, 2 fail.
With the addition, the man test run lists (for me) 7 tests run, 0 failed.

Why the difference in total test count?